docs/50-tutorials/linking-clothes.mdx

```diff
@@ -52,10 +51,21 @@ If the clothing has blendshapes with the same names as the ones on the avatar's
 
 ![image](assets/linking-clothes/230133401-4ab09f83-1597-470d-855f-d40e4ed56909.png)
 
-3. Click the plus on `Skinned meshes to link`, and drag in the renderer object from your clothes. Usually this is the child object inside the clothing prefab that IS NOT the armature. If the clothing prefab contains multiple renderers, you can add them all.
+3. Click the plus on `Skinned meshes to link`, and drag in the renderer/mesh object from your clothes. Usually this is the child object inside the clothing prefab that IS NOT the armature. If the clothing prefab contains multiple renderers, you can add them all.
 
 ![image](assets/linking-clothes/230133576-b00cdb08-c061-428c-aa4f-29651c46b5be.png)
 
 ![image](assets/linking-clothes/230133626-e9c6f7b1-3419-4c22-836f-33fd0cecfe61.png)
 
-4. The debug info box will show the blendshapes which are linked.
+
+:::info
+If your clothes have blendshape names that differ from the base, eg. spelled differently or prefixed, open the **Advanced Options** section and enter the blendshape name on the base and linked mesh.
+
+![image](assets\linking-clothes\Screenshot-2024-02-22-023049.png)
+:::
+
+4. The debug info box will show the blendshapes which are linked. If you linked additional blendshapes, verify that they appear here.
+
+:::warning
+Blendshapes driven by VRChat directly like Visemes, Blinking or Look Up/Down are special and cannot be linked by Blendshape Link as they are not driven through animations.

```

Comment:
Probably need to mention Advanced Visemes

Reply:
I thought about it, but it does have the disadvantage of looking worse than native Visemes.

Reply:
I can probably also remove Look Up/Down from that as they are probably never used when linking meshes. Same probably goes for Blinking, but who knows. Also, there is a component for that so might as well mention it.